lib/gem2rpm/package.rb

```diff
@@ -1,4 +1,6 @@
 require 'delegate'
+# require 'rubygems' to avoid NameError at Ruby 1.8.7.
+require 'rubygems' unless defined?(Gem)
```

Comment:
It is below scenario.

https://travis-ci.org/fedora-ruby/gem2rpm/jobs/129158955
(GEM2RPM_DEBUG=1 is my private environment variable for my debug. Finally I deleted this variable with "git rebase".)

```
$ GEM2RPM_DEBUG=1 rake 
...
 10) Failure:
TestBinGem2Rpm#test_print_current_template [/home/travis/build/fedora-ruby/gem2rpm/test/test_bin_gem2rpm.rb:78]:
Command: /home/travis/build/fedora-ruby/gem2rpm/bin/gem2rpm -T
out: 
err: /home/travis/.rvm/rubies/ruby-1.8.7-p374/lib/ruby/site_ruby/1.8/rubygems/exceptions.rb:9: uninitialized constant Gem (NameError)
	from /home/travis/.rvm/rubies/ruby-1.8.7-p374/lib/ruby/site_ruby/1.8/rubygems/security.rb:7:in `require'
	from /home/travis/.rvm/rubies/ruby-1.8.7-p374/lib/ruby/site_ruby/1.8/rubygems/security.rb:7
	from /home/travis/.rvm/rubies/ruby-1.8.7-p374/lib/ruby/site_ruby/1.8/rubygems/package.rb:43:in `require'
	from /home/travis/.rvm/rubies/ruby-1.8.7-p374/lib/ruby/site_ruby/1.8/rubygems/package.rb:43
	from /home/travis/build/fedora-ruby/gem2rpm/lib/gem2rpm/package.rb:2:in `require'
	from /home/travis/build/fedora-ruby/gem2rpm/lib/gem2rpm/package.rb:2
	from /home/travis/build/fedora-ruby/gem2rpm/lib/gem2rpm.rb:3:in `require'
	from /home/travis/build/fedora-ruby/gem2rpm/lib/gem2rpm.rb:3
	from /home/travis/build/fedora-ruby/gem2rpm/bin/gem2rpm:6:in `require'
	from /home/travis/build/fedora-ruby/gem2rpm/bin/gem2rpm:6
```


The rpm spec file is below thing to prevent this error on RHEL 6.5, with ruby 1.8.7.
"require 'rubygems'" is called before test files are called.

```
ruby -Itest - << \EOF
  require 'rubygems'
  require 'minitest/unit'

  module Minitest; end;

  Minitest::Test = MiniTest::Unit::TestCase
  Dir.glob "./test/**/test_*.rb", &method(:require)
EOF
```